__tests__/app.test.js

```diff
@@ -138,4 +138,63 @@ describe('/api/articles', () => {
             expect(response.body.msg).toBe('Unable to find article')
         })
     })
-})
\ No newline at end of file
+})
+
+ describe('/api/articles/:article_id/comments', () => {
+    test('GET:200 Should respond with an array of objects', () => {
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then((response) => {
+            expect(Array.isArray(response.body)).toBe(true)
+            response.body.forEach((body) => {
+                expect(typeof body).toBe('object')
+            })
+        })
+    })
+    test('GET:200 Should contain the correct properties', () => {
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then((response) => {
+        const comments = response.body
+        comments.forEach((comment) => {
+            expect(comment).toHaveProperty('comment_id')
+            expect(comment).toHaveProperty('votes')
+            expect(comment).toHaveProperty('created_at')
+            expect(comment).toHaveProperty('author')
+            expect(comment).toHaveProperty('body')
+            expect(comment).toHaveProperty('article_id')
```

Comment:
I would expect a solid assertion here that the article_id matches the article you requested. If I asked for comments for article 1 but comments for article 6 this would pass and it shouldn't.